### ISSUE TITLE
Navigation bar - Selected item underline stays default color (orange)

### ISSUE THREAD

**Issue by Robouste** [Alipoodle/GPMDPMidnightAccent#3]
Very nice css, I like it a lot. I have noticed the underline color of the selected item in the navigation bar stays orange. It also happens with built-in dark theme but I think you might want to fix it in your css.

Here is the line to add into `gpmcss.css` : 

`.paper-tabs-1 #selectionBar.paper-tabs { border-color: white; }`

**Comment by Alipoodle** [comment]
Might I ask are you a free account or a Premium account on Google Play Music?

As I'm currently a free user and this is not an issue for me?
![image](https://puu.sh/CrDB9/4fc5f79514.png)

**Comment by Alipoodle** [comment]
``.paper-tabs-1 #selectionBar.paper-tabs {
	background-color:<<FORE_PRIMARY>>;
	border-bottom: 2px solid <<FORE_PRIMARY>>;
}``
Currently I'm achieving it through this. Which indeed is a slightly more complex version of what is actually required from what you have done. I will make the change but otherwise this achieves the same thing? It might be that you need to update your CSS

**Comment by Robouste** [comment]
Hi, I'm indeed using the Premium account. 

In the file I've downloaded from your git the `border-bottom` property was not set for this rule. I've applied your change above, it's more flexible using a variable, thx :)  

I've also added that line:
`.x-scope > div[slot="underline"] { color: <<FORE_SECONDARY>> !important }`

It applies the accent color to the underline for every category in the home page. By default it changes as you scroll but it doesn't really fit the with the dark theme :)
